client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx

```diff
@@ -16,19 +50,24 @@ const PaymentPlan: React.FC< PaymentPlanProps > = ( { isMultiSiteIncompatible }
 				</div>
 			) : (
 				<>
-					<p>Payment plan:</p>
+					<p>{ `${ translate( 'Payment plan' ) }:` }</p>
 
 					<div className="product-lightbox__variants-plan-card">
-						<div className="product-lightbox__variants-grey-label">
-							<span className="product-lightbox__variants-plan-card-price">{ '$4.95' }</span>
-							<span className="product-lightbox__variants-plan-card-month-short">/mo</span>
-							<span className="product-lightbox__variants-plan-card-month-long">/month</span>,
-							billed yearly
-						</div>
-						<div className="product-lightbox__variants-grey-label">
-							<span className="product-lightbox__variants-plan-card-old-price">{ '$9.95' }</span>
-							59% off the first year
+						<div className={ labelClass }>
+							<span className="product-lightbox__variants-plan-card-price ">
+								{ currentPriceValue }
+							</span>
+							<TimeFrame billingTerm={ billingTerm } />
 						</div>
+						{ discountedPrice && (
+							<div className={ labelClass }>
+								<span className="product-lightbox__variants-plan-card-old-price">
+									{ originalPriceValue }
+								</span>
+								{ Math.ceil( ( discountedPrice * 100 ) / originalPrice ) }%
+								{ ` ${ translate( 'off the first year' ) }` }
```

Comment:
That percent off needs to be a part of the translation.

Something like this

```js
translate( '%(percentOff)d\% off the first year' )
```
Second `%` is for the actual percent symbol which is translated in many languages especially RTL languages where there is a different RTL compatible symbol that is used.